LoopKit/Guardrail.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Guardrail.swift
+//  LoopKit
+//
+//  Created by Michael Pangburn on 4/10/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+
+
+public enum SafetyClassification: Equatable {
+    public enum Threshold: Equatable {
+        case minimum
+        case belowRecommended
+        case aboveRecommended
+        case maximum
```

Comment:
Just a question to help me understand. Are these like hard and soft limits, where hard limit cannot be bypassed and soft can be bypassed but only with appropriate authorization/warning?